test/spectests/spectests.cpp

```diff
@@ -27,10 +27,17 @@ fizzy::bytes load_wasm_file(const fs::path& json_file_path, std::string_view fil
         std::istreambuf_iterator<char>{wasm_file}, std::istreambuf_iterator<char>{});
 }
 
+struct test_settings
+{
+    bool skip_validation = false;
+};
+
 class test_runner
 {
 public:
-    void run_from_file(const fs::path& path)
+    explicit test_runner(const test_settings& ts) : settings{ts} {}
+
+    std::tuple<int, int, int> run_from_file(const fs::path& path)
```

Comment:
I would create a `struct` for the result type.

Reply:
Changed.